src/Terrain/terrain.cpp

```diff
@@ -345,7 +345,15 @@ void Terrain::stamp_tile_region(int x_start, int y_start, int z_start, int x_end
     }
 }
 inline void Terrain::stamp_tile_region(Tile_Stamp tStamp, int x, int y){ // unpack Tile_Stamp
-    stamp_tile_region(tStamp.x_start + x*Area_size + Area_size/2, tStamp.y_start + y*Area_size + Area_size/2 , tStamp.z_start, tStamp.x_end + x*Area_size + Area_size/2, tStamp.y_end + y*Area_size + Area_size/2, tStamp.z_end, tStamp.mat, tStamp.elements_can_stamp, tStamp.color_id);
+    stamp_tile_region(tStamp.x_start + x*Area_size + Area_size/2,
+                      tStamp.y_start + y*Area_size + Area_size/2,
+                      tStamp.z_start,
+                      tStamp.x_end + x*Area_size + Area_size/2,
+                      tStamp.y_end + y*Area_size + Area_size/2,
+                      tStamp.z_end,
+                      tStamp.mat,
+                      tStamp.elements_can_stamp,
+                      tStamp.color_id);
```

Comment:
jeez why did you expand the vector

Reply:
Because it needs to be unpacked. There is not much I can do for this. If it ant broke don't fix it.